exercises/practice/yacht/.meta/example.h

```diff
@@ -0,0 +1,27 @@
+#ifndef YACHT_H
+#define YACHT_H
+
+#include <stdint.h>
```

Comment:
I think this include can probably be removed now?
Then I think this PR is good to go!

Reply:
Ah, right! Hadn't had a chance to look at the diff myself yet! This one is gone now as well.